python/python/tests/test_query.py

```diff
@@ -94,6 +94,34 @@ def test_query_builder(table):
     assert all(np.array(rs[0]["vector"]) == [1, 2])
 
 
+def test_query_builder_batches(table):
+    rs = (
+        LanceVectorQueryBuilder(table, [0, 0], "vector")
+        .limit(2)
+        .select(["id", "vector"])
+        .to_batches(1)
+    )
+    assert len(rs) == 2
```

Comment:
Could we assert the type of the returned value?
```suggestion
    assert len(rs) == 2
    assert all(isinstance(item, pa.RecordBatch) for item in rs)
```

Reply:
Thanks, done.